relay-server/src/actors/project_cache.rs

```diff
@@ -209,15 +47,15 @@ impl Message for FetchOptionalProjectState {
 /// Requests to the upstream are performed via `UpstreamProjectSource`, which internally batches
 /// individual requests.
 #[derive(Clone)]
-pub struct UpdateProjectState {
+pub struct RequestUpdate {
```

Comment:
This name was changed for clarity. Note that there is now a new `UpdateProjectState` that performs the actual update in the cache.